content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,256 @@
+    mp4: 'blog/release-post/4.0/Server%20Side%20Batch%20Changes%20'
+  }}
+  loop={true}
+  title="Server-side Batch Changes"
+  caption="Large-scale changes can be run across thousands of repositories with server-side Batch Changes"
+/>
+
+<a href="https://docs.sourcegraph.com/batch_changes/explanations/server_side" className="tw-not-italic tw-flex tw-items-center">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+### <a id="dedicated-sourcegraph-cloud-instances-for-enterprise"></a> ☁️ Dedicated Sourcegraph Cloud instances for enterprise
+
+<br />
+<Badge link="https://docs.sourcegraph.com/cloud" text="Admin" color="violet" size="small" />
+
+####  Sourcegraph Cloud, our single-tenant cloud offering, is now generally available
+After almost a decade of developing self-hosted Sourcegraph, we’ve gained the trust of some of the most sophisticated companies and development teams in the world. Today—after years of building trust and engineering rigor—we are excited to move confidently to the cloud with a highly secure and scalable solution.
+
+Dedicated, single-tenant Sourcegraph Cloud instances are now generally available and the best way for teams to use Sourcegraph. You can read more in our announcement blog [here](/blog/enterprise-cloud), or [sign up for a 30-day trial for your organization](https://signup.sourcegraph.com/).
+
+<a href="https://docs.sourcegraph.com/cloud" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+  
+<br />
+<Badge link="https://docs.sourcegraph.com/cloud" text="Admin" color="violet" size="small" />
+
+#### SOC 2 Type II attestation for Sourcegraph Cloud
+
+We know that your code is one of your most important and sensitive assets. Every component of Sourcegraph was designed with security in mind. As part of our ongoing commitment to security, we recently received our SOC 2 Type II attestation for Sourcegraph Cloud. When you use Sourcegraph, you can be confident that Sourcegraph controls the security and confidentiality of your sensitive data.
+
+Check out our [Security Portal](https://security.sourcegraph.com/) to access the report and learn more about our security program.
+
+<a href="https://security.sourcegraph.com/" className="tw-not-italic tw-flex tw-items-center">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+### <a id="advanced-admin-capabilities"></a> 📈 Advanced admin capabilities
+<br />
+<Badge link="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades" text="Admin" color="violet" size="small" />
+
+#### Upgrade directly to Sourcegraph 4.0 from earlier versions of Sourcegraph
+We know that upgrading Sourcegraph can be a time-intensive process, especially if you fall a few versions behind. Sourcegraph 4.0 supports [multi-version upgrades](https://docs.sourcegraph.com/admin/updates#multi-version-upgrade) so you can upgrade to 4.0 directly from version 3.20 or higher.
```

Comment:
I feel like we're underselling how awesome this is by saying this is only for 4.0. I feel like we should call out that this applies to _any_ version from now on (4.x) etc? That's a way bigger improvement than just a smooth upgrade to 4.0
Maybe something like this?
```suggestion
We know that upgrading Sourcegraph can be a time-intensive process, especially if you fall a few versions behind. Sourcegraph 4.0 supports [multi-version upgrades](https://docs.sourcegraph.com/admin/updates#multi-version-upgrade) so you can upgrade to 4.0 – and any future version – directly from version 3.20 or higher.
```

cc @efritz

Reply:
I like this edit.